fmt/mod.c

```diff
@@ -75,6 +82,71 @@ static const char *valid_tags[][2] = {
 	{NULL, NULL}
 };
 
+enum {
+	WARN_MAXPATTERNS,
```

Comment:
I think we're almost ready to go. Here you can remove unused enum values.

Reply:
Hold my water...

Reply:
Done.